### DISCUSSION TITLE
[F] Add the feature to convert the excel file to pdf.

### DISCUSSION THREAD

**Discussion by arslanalidev** [exceljs/exceljs#2459] [Ideas & Proposals]
## 🚀 Feature Proposal
We should have the option to convert the excel js file into pdf or HTML type. 

## Motivation

When we are generating the excel file, we need to save the files with pdf as well. If we add this feature it will boost our exceljs library.

**Comment by luissantiagodev** [comment]
Current workaround using this lib:
https://www.npmjs.com/package/aspose.cells

**Comment by gera-cameron** [comment]
Upvote here. `aspose.cells` will not work for us...

**Comment by doobybooby** [comment]
Not sure if there was ever a solution for this...
getting an error below when running "npm i apose.cells"

using Mac pro 16

npm ERR! command failed
npm ERR! command sh -c node-gyp rebuild
npm ERR! CXX(target) Release/obj.target/nodejavabridge_bindings/src/java.o
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.0.0
npm ERR! gyp info using node@16.14.0 | darwin | arm64
npm ERR! gyp info find Python using Python version 3.11.4 found at "/opt/homebrew/opt/python@3.11/bin/python3.11"
npm ERR! gyp info spawn /opt/homebrew/opt/python@3.11/bin/python3.11
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',

**Reply by shellking4** [reply]
download and install a viable openjdk and export JAVA_HOME=/usr/lib/jvm/java/openjdk-directory

Then retry
